src/components/NcActionInput/NcActionInput.vue

```diff
@@ -147,6 +155,10 @@ export default {
 					'email'].indexOf(type) > -1
 			},
 		},
+		isNativePicker: {
+			type: Boolean,
+			default: false,
```

Comment:
note: in a future version this should flip to true (breaking change) so we can ease out the custom picker

Reply:
ok, but could i leave it for now like this? It is too risky to make a breaking change now

Reply:
yes. *in a future version* we will flip from false to true. Not now.